src/gateways/FieldGatewayUseGet.js

```diff
@@ -0,0 +1,17 @@
+import { CheckboxFieldFF, ReactFinalForm } from '@dhis2/ui'
+import React from 'react'
+import { dataTest } from '../dataTest'
+import i18n from '../locales'
+
+const { Field } = ReactFinalForm
+
+export const FieldGatewayUseGet = () => (
+    <Field
+        type="checkbox"
+        dataTest={dataTest('gateways-fieldgatewayuseget')}
+        name="useGet"
+        label={i18n.t('Use GET')}
+        component={CheckboxFieldFF}
+        helpText={i18n.t('Not using get results in using POST')}
```

Comment:
This is unclear to me. What are we trying to convey here exactly?

Reply:
https://docs.dhis2.org/master/en/developer/html/webapi_sms.html#

Under `Generic SMS gateway parameters`:

> Http POST method will be used by default. In order to change it and Http GET, user can set useGet flag to true.

Reply:
Maybe we could say something like: `Use GET instead of POST to send data to gateway`

Reply:
Good suggestion, applied in 2485e32